app/src/main/res/menu/account_button.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
```

Comment:
I don't really like menus having just one item. Create one menu file containing all buttons required for activity's menu.